dc-model/src/main/java/de/digitalcollections/model/text/TitleType.java

```diff
@@ -11,6 +11,14 @@ public TitleType() {
   }
 
   public TitleType(String mainType, String subType) {
-    super(mainType, subType);
+    super(mainType, (subType == null || subType.isEmpty() || subType.isBlank()) ? null : subType);
```

Comment:
```suggestion
    super(mainType, (subType == null || subType.isBlank()) ? null : subType);
```
Ist `isEmpty` nicht in `isBlank` mit enthalten?

Reply:
ist gefixed